XamlStyler.Package/app.config

```diff
@@ -0,0 +1,79 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Out of curiosity, why add this file?

Reply:
I think it's a remnant before the migrate all projects from packages.config to PackageReference

Reply:
Gotcha--let's remove it if it isn't important. Same with that extra property that got added to the csproj.